podvm/qcow2/ubuntu/variables.pkr.hcl

```diff
@@ -77,3 +77,13 @@ variable "cloud_provider" {
   type    = string
   default = env("CLOUD_PROVIDER")
 }
+
+variable "is_uefi" {
+  type        = bool
+  default     = false
+}
+
+variable "uefi_firmware" {
+  type    = string
+  default = "/usr/share/edk2/ovmf/OVMF_CODE.cc.fd"
```

Comment:
It seems that Ubuntu 20.04 UEFI firmwares are installed with the ovmf package and the correct default path would be `/usr/share/OVMF/OVMF_CODE.fd`:

```
root@0e43ef089d3b:/# ls /usr/share/OVMF/     
OVMF_CODE.fd     OVMF_CODE.secboot.fd  OVMF_CODE_4M.ms.fd       OVMF_VARS.fd     OVMF_VARS.snakeoil.fd  OVMF_VARS_4M.ms.fd
OVMF_CODE.ms.fd  OVMF_CODE_4M.fd       OVMF_CODE_4M.secboot.fd  OVMF_VARS.ms.fd  OVMF_VARS_4M.fd
```